dev/com.ibm.ws.org.apache.cxf.cxf.core.3.2/src/org/apache/cxf/message/MessageImpl.java

```diff
@@ -68,116 +43,13 @@
-    private static final String HTTP_BASE_PATH = "http.base.path";
-    private static final String SECURITY_CONTEXT = SecurityContext.class.getName();
-    private static final String AUTHORIZATION_POLICY = AuthorizationPolicy.class.getName();
-    private static final String CERT_CONSTRAINTS = CertConstraints.class.getName();
-    private static final String HTTP_SERVLET_RESPONSE = HttpServletResponse.class.getName();
-    private static final String RESOURCE_METHOD = "org.apache.cxf.resource.method";
-    private static final String ASYNC_RESPONSE = "javax.ws.rs.container.AsyncResponse";
-    private static final String SSE_EVENT_SINK = "javax.ws.rs.sse.SseEventSink";
-    private static final Map<String, Integer> KEYMAP;
-    private static final String[] propertyNames = { CONTENT_TYPE, PROTOCOL_HEADERS, QUERY_STRING, AbstractHTTPDestination.HTTP_REQUEST,
-                    AbstractHTTPDestination.HTTP_RESPONSE, REQUEST_PATH_TO_MATCH_SLASH, HTTP_REQUEST_METHOD, INTERCEPTOR_PROVIDERS,
-                    TEMPLATE_PARAMETERS, ACCEPT_CONTENT_TYPE, CONTINUATION_PROVIDER, DESTINATION, OP_RES_INFO_STACK, WSDL_DESCRIPTION,
-                    WSDL_INTERFACE, WSDL_OPERATION, WSDL_PORT, WSDL_SERVICE, REQUEST_URL, REQUEST_URI, PATH_INFO, BASE_PATH,
-                    FIXED_PARAMETER_ORDER, IN_INTERCEPTORS, OUT_INTERCEPTORS, RESPONSE_CODE, ATTACHMENTS, ENCODING,
-                    AbstractHTTPDestination.HTTP_CONTEXT, AbstractHTTPDestination.HTTP_CONFIG, AbstractHTTPDestination.HTTP_CONTEXT_MATCH_STRATEGY,
-                    HTTP_BASE_PATH, ASYNC_POST_RESPONSE_DISPATCH, SECURITY_CONTEXT, AUTHORIZATION_POLICY, CERT_CONSTRAINTS,
-                    AbstractHTTPDestination.SERVICE_REDIRECTION, HTTP_SERVLET_RESPONSE, RESOURCE_METHOD, ONE_WAY_REQUEST, ASYNC_RESPONSE,
-                    THREAD_CONTEXT_SWITCHED, OutgoingChainInterceptor.CACHE_INPUT_PROPERTY, PhaseInterceptorChain.PREVIOUS_MESSAGE,
-                    AbstractHTTPDestination.RESPONSE_HEADERS_COPIED, SSE_EVENT_SINK, REQUESTOR_ROLE, PARTIAL_RESPONSE_MESSAGE,
-                    EMPTY_PARTIAL_RESPONSE_MESSAGE, ENDPOINT_ADDRESS, INBOUND_MESSAGE };
-
-    private static final Object NOT_FOUND = new Object();
-    private static final Integer KEY_NOT_FOUND = Integer.valueOf(-1);
-
-    private Collection<Object> values = null;
-    private Set<String> keySet = null;
-    private Set<Map.Entry<String, Object>> entrySet = null;
-    private static final Object[] NOT_SET_ARRAY = new Object[TOTAL];
-    static {
-        Map<String, Integer> keymap = new HashMap<String, Integer>(TOTAL);
-        for (int i = 0; i < TOTAL; i++) {
-            keymap.put(propertyNames[i], i);
-            NOT_SET_ARRAY[i] = NOT_FOUND;
-        }
-        KEYMAP = Collections.unmodifiableMap(keymap);
-    }
-    //Liberty code change end
-
     // Liberty change - used to avoid resize
     public MessageImpl(int isize, float factor) {
```

Comment:
We can't delete this file (yet) because we still need the `public MessageImpl(int isize, float factor)` constructor - though I plan to contribute that back to the CXF community as soon as possible.